mix.exs

```diff
@@ -90,7 +90,7 @@ defmodule Xebow.MixProject do
       {:phoenix_html, "~> 2.11"},
       {:phoenix_live_dashboard, "~> 0.2.0"},
       {:phoenix_live_reload, "~> 1.2", only: :dev, targets: :host},
-      {:phoenix_live_view, "~> 0.13.0"},
+      {:phoenix_live_view, "~> 0.14.0"},
```

Comment:
:warning: (recommended)

FYI live view is on `0.14.1`. I know symver will automatically pick this up, but it documents our intent in `mix.exs`. For example, if `0.14.2` has a bug and we have to rollback, we'd be able to tell more easily that the app should be ok on `0.14.1` rather than `0.14.0`.

```suggestion
      {:phoenix_live_view, "~> 0.14.1"},
```